warehouse/accounts/views.py

```diff
@@ -816,6 +822,174 @@ def _get_two_factor_data(request, _redirect_to="/"):
     return two_factor_data
 
 
+@view_config(
+    route_name="accounts.verify-organization-role",
+    renderer="accounts/organization-invite-confirmation.html",
+    require_methods=False,
+    uses_session=True,
+    permission="manage:user",
+    has_translations=True,
+)
+def verify_organization_role(request):
+    token_service = request.find_service(ITokenService, name="email")
+    organization_service = request.find_service(IOrganizationService, context=None)
+    user_service = request.find_service(IUserService, context=None)
+
+    def _error(message):
+        request.session.flash(message, queue="error")
+        return HTTPSeeOther(request.route_path("manage.organizations"))
+
+    try:
+        token = request.params.get("token")
+        data = token_service.loads(token)
+    except TokenExpired:
+        return _error(request._("Expired token: request a new organization invite"))
```

Comment:
Double-checking: should this (and the "invalid token" message) say "invite" or "invitation"?